caracal/schema/selfcal_schema.yml

```diff
@@ -422,6 +422,12 @@ mapping:
               - type: int
             required: false
             example: '31, 31'
+          ncpu_img: 
```

Comment:
Since you've started this, might as well add `mem` or `abs-mem` options for wsclean also. No good deed ...

Reply:
...goes unpunished. Indeed. Will do.

Reply:
Have added absmem - note to self, change absmem to abs-mem since it is about to be deprecated.